js/modals.js

```diff
@@ -0,0 +1,85 @@
+
+
+const closePopUp = document.getElementsByClassName('popup_close');
+
+const openPopUp_1 = document.getElementById('popup1_open');
+const openPopUp_2 = document.getElementById('popup2_open');
+const openPopUp_3 = document.getElementsByClassName('popup3_open');
+
+const open_menu = document.getElementById('open_menu')
+
+const popUp1 = document.getElementById('popUp1')
```

Comment:
изменить id на класс в верстке с уникального на общий, далее использовать 
```suggestion
const popUps = document.querySelectorAll('.popUp')
```
и работать как с массивом